less/type.less

```diff
@@ -238,6 +238,75 @@ ol {
   }
 }
 
+.circle-list(@color, @content) {
+  padding: 0;
+  margin: 32px 0;
+  list-style-type: none;
+  li {
+    position: relative;
+    padding-left: 40px;
+    margin-bottom: 16px;
+    strong {
+      color: @color;
+    }
+    &::before {
+      content: @content;
+      color: @color;
+      border: 1px solid @brand-smoke-plus-10;
+      font-family: "TransferWise-IconFont";
+      position: absolute;
+      left: 0;
+      width: 24px;
+      line-height: 22px;
+      margin-right: 16px;
+      font-size: 16px;
+      text-align: center;
+      font-weight: bold;
+      padding: 0 3px;
+      border-radius: 50%;
+    }
+  }
+}
+
+ul.circle-list {
+  &-checked {
+    .circle-list(@brand-green, "\e916")
+  }
+  &-сlosed {
```

Comment:
I'd prefer if the names were 'check' and 'close' so that it's the same name as the icon classes, it just means a little less to remember when using these things.